app/metrics.yaml

```diff
@@ -539,3 +539,35 @@ custom_tab:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+
+activation:
+  identifier:
+    type: string
+    lifetime: ping
+    description: >
+      An hashed and salted version of the Google Advertising ID from the device.
+    send_in_pings:
+    - activation
+    bugs:
+    - 1538011
+    data_reviews:
+    - TODO
+    notification_emails:
+    - fenix-core@mozilla.com
+    expires: "2019-10-01"
+
+  activation_id:
+    type: uuid
+    lifetime: user
+    description: >
+      An alternate identifier, not correlated with the client_id, generated once
+      and only sent with the activation ping.
+    send_in_pings:
+    - activation
+    bugs:
+    - 1538011
+    data_reviews:
+    - TODO
```

Comment:
@liuche how can I make the data review available? Should it be copied over the related Fenix issue or in my implementation bug on bugzilla?

Reply:
@Dexterp37 you can copy the data request here, and I can re-review now that the implementation is also here! I'll flag myself for review here since you mentioned you don't have perms.